addons/website/static/tests/tours/edit_menus.js

```diff
@@ -125,4 +125,33 @@ tour.register('edit_menus', {
         trigger: '#top_menu .nav-item a:contains("Modnar !!")',
         run: () => {}, // It's a check.
     },
+    // Nest menu item from the menu.
+    {
+        content: "Open Pages menu",
+        trigger: '.o_menu_sections a:contains("Pages")',
+    },
+    {
+        content: "Click on Edit Menu",
+        trigger: 'a.dropdown-item:contains("Edit Menu")',
+    },
+    {
+        content: "Drag item into parent",
+        trigger: '.oe_menu_editor li:contains("Contact us") > .ui-sortable-handle',
+        // Menu rows are 38px tall.
+        run: "drag_and_drop [50,38]@.oe_menu_editor li:contains('Home') > .ui-sortable-handle => .oe_menu_editor li:contains('Home') .ui-sortable-placeholder",
+    },
+    {
+        content: "Wait for drop",
+        trigger: '.oe_menu_editor li:contains("Home") ul li:contains("Contact us")',
+        run: () => {}, // It's a check.
+    },
+    clickOnSave,
+    {
+        content: "Menu item should have a child",
+        trigger: '#top_menu .nav-item a.dropdown-toggle:contains("Home")',
+    },
+    {
+        content: "When menu item is opened, child item must be clickable",
+        trigger: '#top_menu .nav-item:contains("Home") ul li a.dropdown-item:contains("Contact us")',
```

Comment:
Not really confortable with last step of the tour being a navigation step, here this is clicking on the contatus menu, starting navigation to /contactus at the same time the tour will end (since the step as found).
The browser / chromeheadless process will be stopped, but there will still be some HTTP calls and / or RPCs being made / processed.
That has always been source of race conditions, even if I think it was somehow improved to make the process wait for all HTTP calls before killing everything.

What I would do here is to make it a check only step (add empty run), and maybe also check the href of the menu is /contactus if you want to ensure that.